staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -315,15 +316,91 @@ func (e *Store) Create(ctx genericapirequest.Context, obj runtime.Object) (runti
 			return nil, err
 		}
 	}
+	if includeUninitialized {
```

Comment:
is there a missing '!'?

Also it seems a bit pointless to modify every Create() method when an outer layer could just sequentially call Create and then have this if block?